boards/arm/mec15xxevb_assy6853/mec15xxevb_assy6853.dts

```diff
@@ -19,8 +19,27 @@
 	};
 
 	aliases {
+		led0 = &led0;
+		led1 = &led1;
+		led2 = &led2;
 		pwm-0 = &pwm0;
 	};
+
+	leds {
+		compatible = "gpio-leds";
+		led0: led_0 {
+			gpios = <&gpiod 14 GPIO_ACTIVE_LOW>;
+			label = "LED 0"; /* LED2 on silkscreen */
```

Comment:
@galak has already reviewed the code and he's the gatekeeper, however I believe the label should be the name as seen on the silkscreen / specified in the datasheet. Also the DTS node name should be better `led2` and only the aliases section should map `led0 = &led2;`

Reply:
The schematic says that GPIO156/LED0 line connects to LED2 (label on silkscreen). I have added comments and changed the label to reflect the ones on silkscreen.